META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
```

Comment:
Why do you need this file? You were supposed to write a simple Math class (you did it) and make it reusable by running mvn clean install - it will put your jar to the maven local repository and you should be able to reuse it in another project or module! No need to make your logger class executable. Only copy/modify existing code and add tests, that's it.